os/drivers/lcd/lcd_dev.c

```diff
@@ -291,6 +292,7 @@ static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 		/* To ensure setpower operation from power saving,
 		 * pm_suspend is calling first. */
 		if (old_power == 0 && new_power > 0) {
+			silent_reboot_lock();
```

Comment:
when lcd on, touch should be on again

Reply:
we don't control touch in lcd driver. it is turn on/of by application

Reply:
okay. thank you for correct